conf/nginx.conf

```diff
@@ -11,5 +11,6 @@ location ^~ __PATH__/ {
   proxy_set_header  Host $host;
   proxy_set_header  X-Real-IP $remote_addr;
   more_set_headers "X-Frame-Options : ALLOW-FROM https://__NEXTCLOUDDOMAIN__ always";
+  add_header Content-Security-Policy "frame-ancestors __NEXTCLOUDDOMAIN__" always;
```

Comment:
Eeh any reason to use `add_header` instead of `more_set_hearders` here ?

Reply:
Yes, because we don't want to overwrite what has been already set for `Content-Security-Policy`, we actually want to add an entry to this header, keeping the options already defined.

Reply:
So I'm 1+ year late about this ...

I don't know if you're more expert than me on how add_header works or if that's the other way around, but from what I know, we specially use more_set_headers instead of add_header because add_header has the counter intuitive behavior of absolutely forgetting about *all* headers defined in higher node of the location tree

Also I doubt that there's any way to concatenate content to the existing CSP header ? Naively I think that you **must** redefine the entirety of the header content ...